service/src/main/kotlin/app/cash/backfila/service/BackfilaServiceModule.kt

```diff
@@ -36,6 +37,7 @@ class BackfilaServiceModule(
     install(BackfilaWebActionsModule())
     install(BackfilaDashboardModule(environment))
     install(ServiceWebActionsModule())
+    install(AdminDashboardModule(environment))
```

Comment:
confused why this wasnt needed before.. is this installed in CashBackfilaService?

Reply:
It's installed by Skim

Reply:
is it going to be broken if it's double installed?

Reply:
will merge, we can figure it out if it does break